### PR TITLE
Add View Command Sequence Diagram to Developer Guide

### PR DESCRIPTION
**Pull request by e0958902**
# Checklist for feature implementation and modification
Closes #<Issue_Number>
- [ ] Added relevant milestone (under milestone tab on the right)
- Implementation
  - [ ] Sources (if any) credited under both relevant section(s) and Developer Guide (Acknowledgement)
  - [ ] JUnit Test(s)
- Documentation
  - [ ] JavaDoc for classes and methods
  - Developer Guide
    - [ ] Design (Can add as a todo statement for later on)
    - [ ] Implementation (Can add as a todo statement for later on)
    - [ ] Relevant User Story present
    - [ ] Non-functional requirement(s) that the feature addresses (if any)
    - [ ] Glossary of the (technical) term(s) used (if any)
    - [ ] Specialised instructions for manual testing (if any)
  - User Guide
    - [ ] Description and usage of feature (if any)
  - Contributions (under `/docs/team/*.md`)
    - [ ] Code/Feature/Enhancement contribution
    - [ ] Developer Guide contribution
    - [ ] User Guide contribution
    - [ ] Misc and team-based admin tasks (if any)

# Checklist for non-implementation related PR
Closes #<Issue_Number>
- [ ] Added relevant milestone (under milestone tab on the right)
- Documentation (Contributions)
  - [ ] Developer Guide contributions (if any)
  - [ ] User Guide contributions (if any)
  - [ ] Review contributions (if any)
  - [ ] Miscellaneous and team-based admin tasks (if any)

# Additional Information
Add sequence diagram for View Command

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -81,6 +81,16 @@ The list medication command extends from Command parent class and contains the f
 then automatically checks off the relevant medication in the DailyMedication list, without user specifying 
 `(-m/-a/-e)` **after** `take -l [index]`.
 
+### View Medication Command
+The view medication command extends from Command parent class and contains the following:
+
+Given below is an example of and how the view command behaves at each step 
+with the help of a the sequence diagram with a usage scenario.
```
Comment (T0nyLin) [inline docs/DeveloperGuide.md:88]: Type "of a the "

Comment (T0nyLin) [reply docs/DeveloperGuide.md:88]: "with a usage scenario" sounds odd

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -81,6 +81,16 @@ The list medication command extends from Command parent class and contains the f
...
+
+![ViewCommand.png](images/ViewCommand.png)
+- Step 1. User initiates MediTracker and done a viewCommand.
```
Comment (T0nyLin) [inline docs/DeveloperGuide.md:91]: "done" should be "does" right?

**Review by T0nyLin — CHANGES_REQUESTED** [review]
Thanks for explaining the process with the SD. Please also make the typo changes.

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (e0958902) [reply docs/DeveloperGuide.md]: Yes, you are right. Good catch. It is resolved in [191f06a](https://github.com/AY2324S2-CS2113T-T09-1/tp/pull/242/commits/191f06a09a9dce0878d824a4640aa440d35e32ca) , but still in progress.

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (e0958902) [reply docs/DeveloperGuide.md]: Yes, you are right. Good catch. It is resolved in  [a7a0246](https://github.com/AY2324S2-CS2113T-T09-1/tp/pull/242/commits/a7a02463300592ac68a0f52c248142e0473ce1ca), but still in progress.

**Review by T0nyLin — APPROVED** [review]
LGTM!

**Pull request merged**